UPDATING.md

```diff
@@ -40,6 +40,16 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Some DAG Processing metrics have been renamed
+
+The following metrics are deprected and won't be emitted in Airflow 2.0:
+
+- `scheduler.dagbag.errors` and `dagbag_import_errors` -- use `dag_processing.import_errors` instead
+- `dag_file_processor_timeouts` -- use `dag_processing.processor_timeouts` instead
+- `collect_dags` -- use `dag_processing.total_parse_time` instead
+- `dag.loading-duration.<basename>` -- use `dag_processing.last_duration.<basename>` instead
+- `dag_processing.last_runtime.<basename>` -- use `dag_processing.last_duration.<basename>` instead
```

Comment:
typo?
both L50 and L51 are the same name now, I wonder could we keep loading-duration or loading_duration and last_runtime, but change from dag to dag_processing? Or anything I missed here...

Reply:
These metrics were _almost_ identicaly.

**`loading-duration`**:
 https://github.com/apache/airflow/blob/23ec78a42403ce9df062cee6371cd28cc0834206/airflow/models/dagbag.py#L411-L413
https://github.com/apache/airflow/blob/23ec78a42403ce9df062cee6371cd28cc0834206/airflow/models/dagbag.py#L388-L393

So loading-duration is the time taken for the `process_file()` call.

**`last_runtime`**:

https://github.com/apache/airflow/blob/f497d1d5aad9d88d6c1cd1dfa28546e7a8c5cb5f/airflow/utils/dag_processing.py#L950-L953
https://github.com/apache/airflow/blob/f497d1d5aad9d88d6c1cd1dfa28546e7a8c5cb5f/airflow/utils/dag_processing.py#L1113-L1114

`last_runtime` was the time taken to by the subprocess that calls `process_file()`, so would always be strictly greater than last-duration.

Having two metrics that were almost but recorded slightly different things seemed more confusing than not to me.

Reply:
I C